### PR TITLE
test for pull request

### LINKED ISSUE #10: yangj want to know how to undo or roll back a merged pull request

### PR DESCRIPTION
**Pull request by yangj**
This is just a test. This will closes #10.

### CODE REVIEW

**Comment by jnowtash** [comment]
comment 1

File: yangj-recommendations.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Los Angeles, CA
+
+### Places to Eat
+Honkaido @ westLA
+
+### Fun things to do
+Hiking at Topanga State park
```
Comment (rlanda123) [inline yangj-recommendations.md:7]: I like it.

**Comment by yangj** [comment]
merge the branch

**Comment by yangj** [comment]
merge test

**Pull request merged**